test cases/linuxlike/14 static dynamic linkage/verify_static.py

```diff
@@ -13,7 +13,8 @@ def handle_common(path):
 def handle_cygwin(path):
     """Handle the Cygwin case."""
     output = subprocess.check_output(['nm', path]).decode('utf-8')
-    if 'I __imp_zlibVersion' in output:
+    if (('I __imp_zlibVersion' in output) or
+       ('D __imp_zlibVersion' in output)):
```

Comment:
```suggestion
    if (('I __imp_zlibVersion' in output) or
        ('D __imp_zlibVersion' in output)):
```

The alignment looks funny here.

Reply:
Ooops, yes.  That's what I get for not using the right editor :)